apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -653,6 +654,14 @@ export class InstallManager {
         }
       }
 
+      if (!isEmpty(packageJson.resolutions)) {
+        tempPackageJson.resolutions = tempPackageJson.resolutions || {};
+        commonPackageJson.resolutions = commonPackageJson.resolutions || {};
+
+        tempPackageJson.resolutions = packageJson.resolutions;
+        commonPackageJson.resolutions = { ...commonPackageJson.resolutions, ...packageJson.resolutions };
```

Comment:
I'm not sure we want all of these hoisted to the common `package.json`. What happens if there are conflicts? It also seems like a valid scenario where only certain packages would want to override package resolution.

Reply:
that's a tricky one, sometimes you might want to have global resolutions to have one instruction. But yes it might case some side effects so I'll remove it. 
How about adding some separate section in `package.json`? Something like `rushGlobalResolutions`?

Reply:
I'd rather not introduce a new package.json field. Will this work if the resolutions are placed in the individual projects' `common/temp` package.jsons?

Reply:
yes, this it totally functional as it is now. The only drawback is that you can't have global resolution settings, i.e. you can control within packages but not across them. Which is not big deal if you provide same resolutions into each package.
The idea was to have control over `common/temp/package.json` as well. That might be useful to enforce resolutions on 3rd parties.